build.gradle

```diff
@@ -17,4 +19,8 @@ buildscript {
 
 task clean(type: Delete) {
     delete rootProject.buildDir
-}
\ No newline at end of file
+}
+
+//repositories {
```

Comment:
Remove commented code

It is considered good practice to leave an empty line at the end of your files